### PR TITLE
Moved "waist" items that aren't at the waist to appropriate strapped location

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Content "Moved waist items that don't make sense to the strapped layer"

#### Purpose of change
Now that sublocations are merged in things can start moving to where they makes sense. I think we should remove the waste item layer entirely but that is outside the scope of this PR. Instead this just takes things that makes sense that are currently waist items and moves them to strapped with the correct sublocations. 

#### Describe the solution
Small code change to allow for sided and unsided sub locations on the same item.

Updated the JSON for the following items to the following locations:

image

#### Testing
Loaded up all the items looking for issues

### CODE REVIEW

File: data/json/items/containers.json [hunk 1]
```diff
@@ -916,16 +917,18 @@
     ],
     "armor_data": {
       "sided": true,
+
```
Comment (kevingranade) [inline data/json/items/containers.json:920]:
```suggestion
```

File: data/json/items/containers.json [hunk 2]
```diff
@@ -916,16 +917,18 @@
...
       "armor": [
         {
           "encumbrance": 2,
           "coverage": 5,
           "covers": [ "leg_l", "leg_r" ],
-          "material": [ { "type": "plastic", "portion_cover": 100, "thickness": 2.0 } ]
+          "specifically_covers": [ "leg_hip_r", "leg_hip_l" ]
```
Comment (kevingranade) [inline data/json/items/containers.json:926]:
```suggestion
          "specifically_covers": [ "leg_hip_r", "leg_hip_l" ],
```

Comment (kevingranade) [reply data/json/items/containers.json:926]: Was going to handle the merge for you, but you un-clicked allow edits.

Comment (bombasticSlacks) [reply data/json/items/containers.json]: Sorry not sure how that happened I've been on mobile so maybe something weird happened with that

**Comment by bombasticSlacks** [comment]
I'll fix this tonight when I have some more time.

**Pull request merged**